pkg/events/client.go

```diff
@@ -0,0 +1,134 @@
+package events
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/google/go-querystring/query"
+	"github.com/workos/workos-go/v2/pkg/workos_errors"
+
+	"github.com/workos/workos-go/v2/internal/workos"
+	"github.com/workos/workos-go/v2/pkg/common"
+)
+
+// ResponseLimit is the default number of records to limit a response to.
+const ResponseLimit = 10
```

Comment:
Is this the default limit we want?

Reply:
Yeah considered using 100 here, but it seems to be the other default response limit across our entire SDK so want to keep it consistent.

Reply:
I believe this limit in general is inconsistent across SDKs (it might be 20 in python), so it's worth examining if they all need to be updated.